php-checks/src/main/java/org/sonar/php/checks/FunctionNameCheck.java

```diff
@@ -41,6 +41,9 @@ public class FunctionNameCheck extends PHPVisitorCheck {
   public static final String DEFAULT = "^[a-z][a-zA-Z0-9]*$";
   public static final String DEFAULT_DRUPAL = "^[a-z][a-z0-9_]*$";
   private static final Pattern patternDrupal = Pattern.compile(DEFAULT_DRUPAL);
+  public static final String DEFAULT_WORDPRESS = "^[a-z][a-z0-9_]*$";
+  private static final Pattern patternWordpress = Pattern.compile(DEFAULT_WORDPRESS);
```

Comment:
```suggestion
  private static final Pattern PATTERN_WORDPRESS = Pattern.compile(DEFAULT_WORDPRESS);
```
Constant should be in UPPERCASE